build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -364,7 +370,7 @@ CONFIG_DEV_CONSOLE=y
 # CONFIG_FDCLONE_DISABLE is not set
 # CONFIG_FDCLONE_STDIO is not set
 CONFIG_SDCLONE_DISABLE=y
-CONFIG_NFILE_DESCRIPTORS=8
+CONFIG_NFILE_DESCRIPTORS=64
```

Comment:
What is reason behind increasing the number of available file descriptors to 64?

Reply:
@sunghan-chang wants the value same as ecode tp1x configs.

Reply:
I don't know how many descriptors are necessary. But 64 is using in our use case.